config/v2/cache/kustomization.yaml

```diff
@@ -1,3 +1,5 @@
+namePrefix: data-science-pipelines-operator-
```

Comment:
Is there a reason why we're adding name Prefixes to individual directories' `kustomization.yaml`'s?

Reply:
Okay nvm, read the description.

Reply:
Opening this comment. This should not be part of this PR but part of a follow-up issue to fix later

Reply:
@rimolive Due to the `namePrefix` already being in the base `kustomization.yaml` file, the ConfigMaps and Secrets were inadvertently getting the same prefix which didn't resolve this particular issue. I had no choice but to move all these `namePrefix`'s to the respective `kustomization.yaml`'s to prevent the ConfigMap and Secrets from inheriting the same. Also I researched if there was a possibility to exclude such namePrefix's for said types, but it wasn't designed to skip unfortunately.

Reply:
@amadhusu you are right, but @rimolive 's piont is valid, let me propose a different solution, add these configmap/secrets here: https://github.com/opendatahub-io/data-science-pipelines-operator/tree/dspv2/config/overlays/make-v2deploy 

and not part of the `v2` kustomize build, so the annotations are not inherited. This is also useful to separate these out, because we will likely want to remove these later (we don't want to add tekton resources, it's more or less to get the deployment in a working state right now), since these are tekton resources and it's unclear right now why they are not included by openshift-pipelines.

Reply:
Oh I missed the fact, there is another base `kustomization.yaml` which supercedes the `v2` kustomize build. That makes more sense, I am modifying the PR to include the ConfigMaps and Secrets here instead which makes more sense. Thanks @HumairAK .